specs/posix/sudoers.table

```diff
@@ -0,0 +1,7 @@
+table_name("sudoers")
+description("Rules for running commands as other users via sudo.")
+schema([
+    Column("header", TEXT, "Symbol for given rule"),
+    Column("rule_details", TEXT, "Rule definition")
```

Comment:
I agree that there could be more utility with more columns, but the tricky part is determining the correct columns such that all entries are able to be split up. For example, if we were to define "Terminals", "Users", and "Commands", this would work fine for the (User) and (Group) type rules, but doesn't really make sense for (Defaults) or (Aliases).